test/e2e/utils.go

```diff
@@ -414,3 +414,14 @@ func createSecret(secretName, secretNamespace string, secretData map[string][]by
 	WaitForSecret(secretName, secretNamespace)
 	return createdSecret
 }
+
+func deletePersistentVolumeClaims(namespace string) {
+	pvcs, err := fw.KubeClient.CoreV1().PersistentVolumeClaims(kafkaNamespace).List(metav1.ListOptions{})
```

Comment:
What happens to the volumes? Are they deleted automatically?

Reply:
Normally they are deleted when the namespace is deleted, but not when the Kafka instance is deleted.  Deleting them in this function just covers some edge cases where the namespace for a test may not get deleted because a subsequent test panics, or something of that sort.  

This has caused us a few problems on internal CI.

Reply:
@jpkrohling Yes, if you delete the pvc the pv is deleted along with it.